Windows.md

```diff
@@ -17,29 +17,70 @@
 
 ## Requirements
 
-Apache NiFi MiNiFi C++ has been built on Window Server 2016, 2019, and Windows 10 operating systems. Since the project is CMAKE focused we suggest building via
-Visual Studio 2017 or our ms_build.bat script. In order to build the MSI, please install the [WiX Toolset](https://wixtoolset.org/)
+Apache NiFi MiNiFi C++ has been built on Window Server 2016, 2019, and Windows 10 operating systems. The project is CMake focused we suggest building via
+Visual Studio 2017 or our `win_build_vs.bat` script.
 
-The project previously required OpenSSL to be installed. If you follow our build procedures, below, you will not need to install that dependency. Further, any MSI
-distributable requires that systems install the [Visual Studio 2010 redistributables](https://www.microsoft.com/en-us/download/details.aspx?id=26999&irgwc=1&OCID=AID681541_aff_7593_312327&tduid=(ir__2dz2dkkuookfr3a22jsv1zgwwn2xmb9vrhvzwmdq00)
+The project previously required OpenSSL to be installed. If you follow our build procedures, below, you will not need to install that dependency.
+
+### Required software
+
+ - Visual Studio 2017
+ - [CMake](https://cmake.org/download/)
+ - [Git](https://git-scm.com/download/win)
+ - (Optional) [WiX Toolset](https://wixtoolset.org/releases/) (only for building the MSI)
+ - (Optional) JDK (only for JNI support)
+
+### JNI support
+Though the project is written in C++, JNI functionality supports running Java processors stored in NiFi Archives. These can be run
+in a much smaller memory footprint and consumer fewer resources. If your systems do not support Java or you do not want a JDK installed, please use non-JNI builds.
```

Comment:
consumer -> consume

Reply:
Nice catch, this was originally in the file, I just moved it and didn't verify it properly.